src/main/java/frc/robot/subsystems/InterfaceSubsystem.java

```diff
@@ -114,108 +178,22 @@ private void executeDrive(
     CommandScheduler.getInstance().schedule(drive_command);
   }
 
-  public void driveToLoc(InterfaceExecuteMode loc, InterfaceActionCmd stuff) {
-    boolean isRight = false;
+  public void driveToLoc(
+      InterfaceExecuteMode loc, InterfaceActionCmd stuff, int lvl, boolean isRight) {
+    if (lvl != -1) level = lvl;
```

Comment:
Little ugly with the variable naming here, to work around the member (`level`). Would be good to document what `lvl` is and its usage here.